.github/workflows/depend.yml

```diff
@@ -0,0 +1,14 @@
+name: depend
+on:
```

Comment:
I believe we want to have pull request as a trigger.

Reply:
Made changes ,Now the dependent part is not triggering.